### PR TITLE
chore: fix network

### PR DESCRIPTION
**Pull request by kumaraditya303**
Will add tests later

### CODE REVIEW

**Comment by kumaraditya303** [comment]
cc @mxschmitt

File: generated_interfaces.go [hunk 1]
```diff
@@ -1364,6 +1364,14 @@ type Page interface {
 // If request gets a 'redirect' response, the request is successfully finished with the 'requestfinished' event, and a new
 // request is  issued to a redirected url.
 type Request interface {
+	// An object with all the request HTTP headers associated with this request. The header names are lower-cased.
+	AllHeaders() (map[string]string, error)
+	// An array with all the request HTTP headers associated with this request. Unlike Request.allHeaders(), header
+	// names are NOT lower-cased. Headers with multiple entries, such as `Set-Cookie`, appear in the array multiple times.
+	HeadersArray() ([]map[string]string, error)
```
Comment (mxschmitt) [inline generated_interfaces.go:1371]: This is not the type which we have upstream. Upstream we have []struct{name: string, value: string}

File: generated_interfaces.go [hunk 2]
```diff
@@ -1409,10 +1417,21 @@ type Request interface {
 
 // `Response` class represents responses which are received by page.
 type Response interface {
+	// An object with all the response HTTP headers associated with this response.
+	AllHeaders() (map[string]string, error)
+	// An array with all the request HTTP headers associated with this response. Unlike Response.allHeaders(), header
+	// names are NOT lower-cased. Headers with multiple entries, such as `Set-Cookie`, appear in the array multiple times.
+	HeadersArray() ([]map[string]string, error)
```
Comment (mxschmitt) [inline generated_interfaces.go:1424]: same as in the request

**Review by mxschmitt — APPROVED** [review]

File: generated_interfaces.go (hunk 1 above)
Comment (kumaraditya303) [reply generated_interfaces.go:1375]: Ok will be fixed once will add tests

**Pull request merged**